docs/view_components/welcome.rst

```diff
@@ -0,0 +1,38 @@
+linking ************
+Welcome page
+************
+
+A welcome window can be seen in almost every software/website in production. The welcome window should serve as an orientation point for the user and let them know, in as brief words (and other audiovisual content) as possible, what to expect from the tool. It is also the launch point from where the user begins their energy systems simulation.
+
+The welcome page is to be implemented as an optional pop-up that springs up on the landing page when the user launches the tool. In this section, a structure for the welcome page is proposed, subject to further discussion with the stakeholders.
+
+Attributes
+----------
+
+The following are the view-components available to the user in the welcome pop-up on the landing window immediately after launching the software for the first time:
+
+#. **Welcome message on the pop-up**
+    * A text message 'Welcome to open_plan, the free and open-source energy systems planning and simulation tool based on the *OEMOF* framework and *energy cells* concept
```

Comment:
The description should just say " A welcome text", not necessarily providing the text here. The text's content will be thought though in a second step: if we decide not to have a Welcome window after consultation, you effort to find a text are then in vain.

Reply:
Since I already wrote it, I will add 'A welcome text along the lines of' to the sentence and leave it as it is.